config.php

```diff
@@ -135,7 +135,13 @@ static function xdebugOutputFormat() {
      * Directory to search for trace files
      */
     static function xdebugOutputDir() {
-        $dir = ini_get('xdebug.profiler_output_dir');
+	$version = phpversion('xdebug');
+	if ($version >= '3.0.0') {
```

Comment:
I don't believe this will work, it needs to use `version_compare`, but I also don't believe it's necessary either. Check for `output_dir` and if it doesn't exist, use `profiler_output_dir` instead

Reply:
@tomjn

> I don't believe this will work, it needs to use `version_compare`, but I also don't believe it's necessary either. Check for `output_dir` and if it doesn't exist, use `profiler_output_dir` instead

I saw you updated the code, closing this PR